package.json

```diff
@@ -40,5 +43,30 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "husky": "^4.3.8",
+    "lint-staged": "^10.5.4",
+    "prettier": "^2.2.1",
+    "pretty-quick": "^3.1.0",
+    "ts-jest": "^26.5.0"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.ts?$": "ts-jest"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!(jsoneditor-react))/"
+    ]
+  },
+  "lint-staged": {
+    "*.js": "eslint"
+  },
+  "husky": {
+    "hooks": {
```

Comment:
I am not sure how to make this work in CLI. I tried running:
```
$ bash .git/hooks/pre-commit
```
also changing the code to [issue](https://github.com/paralink-network/paralink-ui/pull/13#discussion_r571461644) mentioned in the PR to make it fail and the committing. Nothing happened. See [here](https://typicode.github.io/husky/#/) for auto git hook installation.

If possible, please provide instructions on how to setup pre-commit (README), so it will work from CLI out of the box (both manually running the pre-commit/husky script as well as automatically running the appropriate git hooks on commit). Assume that not everyone is using VSCode and that bash workflow should be the ground truth. This is also needed for setting up CI.

Reply:
Yeah normally this should directly run at "commit" time. I'll add some README, I believe what's happening is that you might not have installed husky ? ( Wondering if you did npm i )
```bash .git/hooks/pre-commit``` from the root repo should normally work, do the test.

If this still occurs, I can help dig into it more

Reply:
I did run `npm i` and verified that `husky` was installed. It printed the lines: "Installing the hooks". For your convenience I cloned a fresh repo. It did not work as well, however if I removed all my default hooks, reinstalled `husky` it copied the hooks. It [appears](https://github.com/typicode/husky/issues/171#issuecomment-390796547) this is an open issue with `husky`, which frankly is a bit strange that they do not append to the existing file.

Reply:
Yeah saw that actually and ran myself into the issue when trying to reproduce. I added some steps for now to debug ( basically removing the hooks and reinstalling worked for me ) This probably avoids the default ones